activedata_etl/imports/file_mapper.py

```diff
@@ -142,7 +142,13 @@ def find_best(files, complain):
                 return {"name": filename}
             suite_names = SUITES.get(task_cluster_record.suite.name, {task_cluster_record.run.suite.name})
 
-            filename = filename.split(' line ')[0].split(' -> ')[0].split('?')[0].split('#')[0]  # FOR URLS WITH PARAMETERS
+            split_fname = filename.split(' line ')[0].split(' -> ')
+            fname_to_use = split_fname[0]
+            if len(split_fname) > 1:
+                # GET LAST FILE OF LIST (THE ONE THAT IS COVERED)
+                split_fname = fname_to_use[-1]
```

Comment:
is this line supposed to be `fname_to_use = split_fname[0]`?

Reply:
maybe this can be written as

```python
fname_to_use = filename.split(' line ')[0].split(' -> ')[-1]
```

or maybe 

```python
filename = filename.split(' line ')[0].split(' -> ')[-1].split('?')[0].split('#')[0]  # FOR URLS WITH PARAMETERS
```